GenderPayGap.WebUI/Views/Components/ReportOverview/ReportOverviewSectionsViewModel.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace GenderPayGap.WebUI.Views.Components.ReportOverview
 {
-    public class ReportOverviewSectionsViewModel : GovUkViewModel
+    public class ReportOverviewSectionsViewModel
```

Comment:
I know it's not because of you that it's here, but can we move this to the Models folder and fix its namespace?